build.gradle

```diff
@@ -50,3 +50,9 @@ subprojects {
         }
     }
 }
+
+tasks.register("printVersion") {
+    doLast {
+        println(project.property("VERSION_NAME"))
```

Comment:
nit: mmm I'm not 100% but I think best practice (to make the most of caching) would be to write it to a file and return the file location, then your other code reads from that file

Reply:
We don't want to cache this, this is only going to be used from CI to know whether this version is a snapshot or not (so we can avoid auto publishing to maven central if it's a snapshot).